src/app/games/lodestone/LodestoneRules.ts

```diff
@@ -225,14 +225,16 @@ export class LodestoneRules extends Rules<LodestoneMove, LodestoneState, Lodesto
         const moved: Coord[] = [];
         const directions: readonly Ordinal[] = orientation === 'diagonal' ? Ordinal.DIAGONALS : Ordinal.ORTHOGONALS;
         for (const direction of directions) {
-            for (let coord: Coord = lodestone.getNext(direction); // eslint-disable-next-line indent
-                 state.isOnBoard(coord); // eslint-disable-next-line indent
-                 coord = coord.getNext(direction)) {
+            for (
+                let coord: Coord = lodestone.getNext(direction);
+                state.isOnBoard(coord);
+                coord = coord.getNext(direction))
+            {
```

Comment:
vomissure galactique
mais j'vois pas comment le faire plus court pour que ça fit une ligne :paintbrush:

Reply:
un WHILE :D
such complexe and new technology :D